LOGGING/src/ILogger/ApplicationInsightsLogger.cs

```diff
@@ -96,7 +96,7 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                         this.PopulateTelemetry(traceTelemetry, state, eventId);
                         if (exception != null)
                         {
-                            traceTelemetry.Properties.Add("ExceptionMessage", exception.Message);
+                            traceTelemetry.Properties.Add("ExceptionMessage", exception.ToInvariantString());
```

Comment:
should we keep this as is, and add a new property `ExceptionStackTrace` or similar?

Reply:
Reporting both ExceptionMessage and ExceptionStackTrace makes it more aligned with OpenTelemetry conventions.
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/exceptions.md

Reply:
I am fine with doing that!

Reply:
Done!.